includes/controllers/class-attribute.php

```diff
@@ -183,7 +183,7 @@ function( $name ) {
 			$taxonomy = hp\prefix( $model_name . '_category' );
 
 			if ( taxonomy_exists( $taxonomy ) ) {
-				wp_set_post_terms( $post->ID, [ absint( $request->get_param( hp\prefix( 'categories' ) ) ) ], $taxonomy );
+				wp_set_post_terms( $post->ID, $request->get_param( hp\prefix( 'categories' ) ), $taxonomy );
```

Comment:
Please always sanitize input, e.g. this code doesn't ensure that there's an array of IDs (positive numbers) and accepts an array of strings or other values via POST. Just passing a raw request parameter to a WP function is not ok, even if we believe that they care about SQL injections on their side

The general rule is "sanitize input, escape output", it ensures that even if the dangerous value is saved (e.g. JS script string) in a place where it's output it'll be escaped anyway and will be displayed as a plain text instead of output as HTML tag